### PR TITLE
.

### PR DESCRIPTION
**Pull request by waifwu**

### CODE REVIEW

File: script/Arcueid/Day 1/01_00_ARC01_3_1.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ There's no need to go rushing off.
 [sha:b4fd03656e7816a6a9334991390359221722c0c4]{
 -- Page 10007, Offset 1692.
 -- 一分一秒の単位で急いだところで、事態が好転する話でもないのだし。
-It's not as if the situation is going to improve if I spend all my time hurrying about.
+It's not as if the situation is going to improve if I spend all my time hurrying about it.
```
Comment (yk1512) [inline script/Arcueid/Day 1/01_00_ARC01_3_1.txt:14]: revert

File: script/Arcueid/Day 1/01_00_ARC01_5.txt [hunk 2]
```diff
@@ -227,7 +227,7 @@ I had prepared for this situation, but I still gulp and look around the vast lob
 [sha:0a6368ceec81a2a16433c2c6447d0d4312e23aac]{
 -- Page 262, Offset 1829.
 -- どこか違う国、違う時代にタイムスリップしてしまった錯覚に襲われる。屋敷の内装はあまりにも時代がかっていて、その空気さえ違っていた。
-I get the striking feeling that I've been transported to a different country in a different time. The interior of the mansion is so dated that even the atmosphere is different.
+I get the striking feeling that I've been transported to a different country at a different time. The interior of the mansion is so dated that even the atmosphere is different.
```
Comment (yk1512) [inline script/Arcueid/Day 1/01_00_ARC01_5.txt:230]: revert

File: script/Arcueid/Day 1/01_00_ARC01_6.txt [hunk 3]
```diff
@@ -74,7 +74,7 @@ I can't help but feel embarrassed as she serves me politely with a smile on her
 [sha:613a57ad8684b307f9b9052772b0faaea820c009]{
 -- Page 301, Offset 1914.
 -- 有間の家での生活云々ではなく、きっと根が小市民なのだ、俺は。
-It's not because of my life at the Arima house and such. I'm just lower middle class at heart.
+It's not because of my life at the Arima house and such. I'm just a lower middle class at heart.
```
Comment (yk1512) [inline script/Arcueid/Day 1/01_00_ARC01_6.txt:77]: revert

File: script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 4]
```diff
@@ -167,7 +167,7 @@ I'm not that kind of person, and to be honest, it's so far above me that I feel
 [sha:05d18f939a9f7a54d518fb37da49a24637c1b093]{
 -- Page 532, Offset 2510.
 -- 「そうは言うけど、俺は今日まで普通に生きてきた身なんだ。
-"You say that, but up until today I was living a normal life.
+"You say that but up until today, I was living a normal life.
```
Comment (yk1512) [inline script/Arcueid/Day 1/01_00_ARC01_9B.txt:170]: revert

File: script/Arcueid/Day 2/02_00_ARC02_3_1.txt [hunk 5]
```diff
@@ -11,7 +11,7 @@ I promised Akiha that I'd be back before sunset, after all.
 [sha:d0fa7ff4a138ef1278b9533c12bda648ee9a08e2]{
 -- Page 175, Offset 3055.
 -- 連続殺人事件なんてニュースを鵜呑みにする訳ではないが、人生、何があるか分からない。
-It's not like I'm afraid of the news about the serial killings,
+It's not like I'm afraid of the news about serial killings,
```
Comment (yk1512) [inline script/Arcueid/Day 2/02_00_ARC02_3_1.txt:14]: revert

File: script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 6]
```diff
@@ -252,7 +252,7 @@ The girl walks away.
 [sha:4c2d800161df982ff0d8ed14f18c2ae88ad31566]{
 -- Page 921, Offset 8108.
 -- 去り際、なんとなくではあるが、俺よりあの子の方が何倍も体調が悪かったのでは、と思ってしまった。
-As she left, I somehow got the feeling that her physical condition was far more dire than my own.
+As she left, I somehow got the feeling that her physical condition was far direr than my own.
```
Comment (yk1512) [inline script/Arcueid/Day 4/04_00_ARC04_6_1.txt:255]: revert

File: script/Arcueid/Day 6/06_00_ARC06_1B.txt [hunk 7]
```diff
@@ -64,7 +64,7 @@ It's a cane... no, a crutch.
 [sha:c1d93f81d15ec06967da589b5feec6a42a9f281f]{
 -- Page 43, Offset 14027.
 -- 一縷の望みは言い過ぎ……と笑いかけて、翡翠の息がかすかにあがっている事に気がついた。
-'Sliver of hope' is a bit of an exaggeration... I smile at the turn of phrase before noticing that Hisui's breathing is a little elevated.
+'Sliver of hope' is a bit of an exaggeration... I smile at the turn of the phrase before noticing that Hisui's breathing is a little elevated.
```
Comment (yk1512) [inline script/Arcueid/Day 6/06_00_ARC06_1B.txt:67]: revert

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 8]
```diff
@@ -344,7 +344,7 @@ Didn't I turn a blind eye so that I wouldn't be aware of that fact?
 [sha:4589f3963177854485aef4956e9eaa3e473d752c]{
 -- Page 929, Offset 8126.
 -- 遅れを取り戻すようにナイフを取り出す。
-I whip out my knife, as though trying to make up for lost time.
+I whip out my knife, as though trying to make up for the lost time.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_1.txt:347]: revert

**Review by Avuxo — APPROVED** [review]
approval for just the 1 comma

**Pull request merged**